tsconfig.json

```diff
@@ -15,7 +16,7 @@
     "checkJs": false,
     "forceConsistentCasingInFileNames": true,
     "esModuleInterop": true,
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
Revert this change.  Running the app causes it to change anyway.  IntelliJ for some reason changes it when running tests.

Reply:
yea everytime i change it back it changes back to this, ill revert it

Reply:
You need to set the env var `DISABLE_NEW_JSX_TRANSFORM=true` - our npm scripts do this for you.

Reply:
Ah.  I will try setting that in the IDE.

Reply:
This should be undone

Reply:
Agreed